.travis.yml

```diff
@@ -2,6 +2,7 @@ dist: xenial
 language: node_js
 node_js:
   - "10"
+  - "12"
```

Comment:
Do we still support 11 or versions earlier than 10?

Reply:
It should work on earlier versions but internally we only use node 10 and starting next quarter node 12, so those are the versions we're most concerned about and the ones chosen to run tests.